cli/internal/fs/path.go

```diff
@@ -145,7 +145,7 @@ func (ap AbsolutePath) RelativePathString(path string) (string, error) {
 	return filepath.Rel(ap.asString(), path)
 }
 
-// Symlink implements os.Symlink(target, ap) for absolute path
+// Symlink implements os.Symlink(target) for absolute path
```

Comment:
I think this was correct before. The underlying call is in fact `os.Symlink(target, ap.asString())`, so `os.Symlink(target, ap)` where `ap` is the `AbsolutePath` matches.